### ISSUE TITLE
Storage should get more specs

### ISSUE THREAD

**Issue by RobTerwel** [quintel/mechanical_turk#88]
Post-deploy, continuation of #84 and #87.

- [ ] check if putting P2H first in the 'flexibility order' decreases the hot water demand
- [ ] check if putting P2G first in the 'flexibility order' decreases the natural gas demand

This probably requires adjustments in the converter API, particularly in <a href="https://github.com/quintel/etengine/blob/master/app/models/qernel/converter_api/attributes.rb"a>attributes.rb<a/> . By adding a `:flexibility_order_position` attribute, we can create a `flexibility_order` gquery which `mech_turk` can then ask for.

More relevant tests to be added!

**Comment by ChaelKruip** [comment]
- [ ] check whether the excess electricity is conserved (add up all uses for excess including losses)

**Comment by RobTerwel** [comment]
Putting this on hold as this requires (even) further development of the mechanical turk. 

Even if the `flexibility_order_position` can be queried, that doesn't mean that we can *set* it. The tests listed above requires us to be able to change the flexibility order within the mechanical turk. Since the input of this order is not a classic input statement but a table in ETModel which is manipulated by the user, we cannot simply change it in the way we usually do. That should be done first before these tests should be implemented.

**Comment by ChaelKruip** [comment]
:+1:
